### PR TITLE
dockdoor 1.1.4 (new cask)

### PR DESCRIPTION
**Pull request by ejbills**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/d/dockdoor.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "dockdoor" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/ejbills/DockDoor/releases/latest/download/DockDoor.dmg"
```
Comment (khipp) [inline Casks/d/dockdoor.rb:5]:
```suggestion
  version "1.1.4"
  sha256 "f4efa9b1dc8f06228ffe2e6e8ca23fe56cea2e45aa77d864049347a5682899c1"

  url "https://github.com/ejbills/DockDoor/releases/download/v#{version}/DockDoor.dmg"
```

**Comment by khipp** [comment]
Thank you for your submission, @ejbills! Please also try to squash your commits.

File: Casks/d/dockdoor.rb (hunk 1 above)
Comment (ejbills) [reply Casks/d/dockdoor.rb:5]: Quick question: Is this change required? If so, would it be permissible to set up a GitHub Action in my DockDoor repository that automatically opens a PR into this repository to update this information whenever a new DockDoor release is published?

Comment (khipp) [reply Casks/d/dockdoor.rb:5]: We use a checksum whenever possible. Once you publish a new release, `brew bump --open-pr dockdoor` will update the `version` and `sha256` stanzas based on the tag and binary, and create a PR.

**Comment by ejbills** [comment]
Squashed, should be good to go. Thanks Klaus for the clarification.

**Review by khipp — APPROVED** [review]
Thank you, @ejbills!

**Pull request merged**